azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/KnownWindowsVirtualMachineImage.java

```diff
@@ -16,8 +16,10 @@
     /** Windows Server 2012 R2 Data center. */
     WINDOWS_SERVER_2012_R2_DATACENTER("MicrosoftWindowsServer", "WindowsServer", "2012-R2-Datacenter"),
     /** Windows Server 2016 technical preview with container support. */
+    @Deprecated
```

Comment:
in addition to the annotation, we can also put in a Javadoc tag `@deprecated` and say what to do instead (e.g. see ApplicationGatewayBackendHttpConfiguration.java)

Reply:
done, please review.